example-projects/actix-web-example/src/main.rs

```diff
@@ -1,5 +1,5 @@
 use actix_web::{get, post, App, HttpServer};
-use cloudevents::{Event, EventBuilder, EventBuilderV10};
+use cloudevents::{Event, EventBuilder, EventBuilderV10, test_data::*};
```

Comment:
I would never expect to see this outside of a `#[cfg(test)]` module.

Reply:
> Seeing the example, I'm now having 2nd thoughts about adding a `test` feature. I like sharing the fixtures between unit and integration tests within this repo, but consumers of this library should have their own fixtures, I think.

I see what you guys are saying and it makes sense. Although I hoped func could have used them as well. If we want to keep it SDK only, I guess not making the module public would help.